src/builder.rs

```diff
@@ -1,7 +1,10 @@
 use crate::config::{GlobalPassOption, JscTarget, ModuleConfig};
-use atoms::JsWord;
-use common::{chain, errors::Handler, fold::and_then::AndThen, SourceMap};
-use ecmascript::{
+use either::Either;
+use hashbrown::hash_map::HashMap;
+use std::sync::Arc;
+use swc_atoms::JsWord;
+use swc_common::{chain, errors::Handler, fold::and_then::AndThen, SourceMap};
+use swc_ecmascript::{
```

Comment:
I want swc to reexport subcrate and export it like `swc::common`, `swc;;ecmascript::ast`. Also, I prefer common over swc_common.

Reply:
sure thing - how can that be done? :D

Reply:
Oh.. Renaming is not possible at here without extern crate. Sorry for my mistake.

I thought it is possible because `extern crate`-s exist before changing to toml-based renaming